docs/reference/overview/excel-add-ins-reference-overview.md

```diff
@@ -8,56 +8,38 @@ localization_priority: Priority
+Review other articles in this section of the documentation for guidance about using the Excel JavaScript API to implement various types of functionality in your add-in. For example, learn about working with [events](../../excel/excel-add-ins-events.md), [charts](../../excel/excel-add-ins-charts.md), [ranges](../../excel/excel-add-ins-ranges.md), [tables](../../excel/excel-add-ins-tables.md), [worksheets](../../excel/excel-add-ins-worksheets.md), and more. Also here in this section of the documentation, you'll find guidance about Excel JavaScript API concepts such as [coauthoring in Excel add-ins](../../excel/co-authoring-in-excel-add-ins.md), [data validation](../../excel/excel-add-ins-data-validation.md), [error handling](../../excel/excel-add-ins-error-handling.md), and [performance optimization](../../excel/performance.md). See the table of contents for the complete list of available articles.
 
-- [Filter](/javascript/api/excel/excel.filter): Represents an object that manages the filtering of a table's column.
+For hands-on experience using the Excel JavaScript API to access objects in Excel, complete the [Excel add-in tutorial](../../tutorials/excel-tutorial.md). 
 
-- [NamedItem](/javascript/api/excel/excel.nameditem): Represents a defined name for a range of cells or a value.
-  - [NamedItemCollection](/javascript/api/excel/excel.nameditemcollection): A collection of the **NamedItem** objects in a workbook.
+For detailed information about the Excel JavaScript API object model, see the [Excel JavaScript API reference documentation](/javascript/api/excel).
 
-- [Binding](/javascript/api/excel/excel.binding): An abstract class that represents a binding to a section of the workbook.
-  - [BindingCollection](/javascript/api/excel/excel.bindingcollection): A collection of the **Binding** objects in a workbook.
+## Try out code samples in Script Lab
 
-## Excel JavaScript API requirement sets
-
-Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For detailed information about Excel JavaScript API requirement sets, see the [Excel JavaScript API requirement sets](../requirement-sets/excel-api-requirement-sets.md) article.
-
-## Excel JavaScript API reference
-
-For detailed information about the Excel JavaScript API, see the [Excel JavaScript API reference documentation](/javascript/api/excel).
+Use [Script Lab](../../overview/explore-with-script-lab.md) to get started quickly with a collection of built-in sample snippets that show how to complete certain tasks with the API. You can run the samples in Script Lab to instantly see the result in the task pane or document, examine the samples to learn how the API works, and even use sample snippets as the basis for prototyping functionality of your own add-in.
```

Comment:
Second sentence: 
- Consider changing "document" to "worksheet" to make it more Excel-centric.
- Consider "and even use sample snippets as the basis for prototyping functionality of your own add-in." -> "and even prototype your add-in with our samples." It's more concise and dramatic.